### PR TITLE
[Misc] Blackhole attributes can now be packed.

### PR DESCRIPTION
**Pull request by 3Hren**
Also I've added pack/unpack traits for `std::pair` just for convenience.

### CODE REVIEW

File: include/cocaine/traits/tuple.hpp [hunk 1]
```diff
@@ -292,6 +292,30 @@ struct type_traits<
     }
 };
 
+// Pair serialization
+
+template<typename T, typename U>
+struct type_traits<std::pair<T, U>> {
```
Comment (kobolog) [inline include/cocaine/traits/tuple.hpp]: Why wouldn't you simply inherit from `type_traits<std::tuple<T, U>>`?

File: src/service/logging.cpp [hunk 2]
```diff
@@ -21,10 +21,12 @@
 #include "cocaine/detail/service/logging.hpp"
 
 #include "cocaine/context.hpp"
+
```
Comment (kobolog) [inline src/service/logging.cpp]: Why this whitespace appeared all of a sudden?

File: include/cocaine/traits/tuple.hpp (hunk 1 above)
Comment (3Hren) [inline include/cocaine/traits/tuple.hpp]: Done.

File: src/service/logging.cpp (hunk 2 above)
Comment (3Hren) [inline src/service/logging.cpp]: Done.

**Pull request closed without merging**